decidim-meetings/app/services/decidim/meetings/meeting_iframe_embedder.rb

```diff
@@ -36,14 +36,15 @@ def embeddable?
       def embed_code(request_host)
         return nil if parsed_online_meeting_uri.nil?
 
-        %(
-<iframe
-  allow="camera; microphone; fullscreen; display-capture; autoplay"
-  loading="lazy"
-  src="#{embed_transformed_url(request_host)}"
-  style="height: 100%; width: 100%; border: 0px;"
-></iframe>
-        )
+        <<~HTML
+          <div class="disabled-iframe">
+            <!-- <iframe
+              allow="camera; microphone; fullscreen; display-capture; autoplay"
+              loading="lazy"
+              src="#{embed_transformed_url(request_host)}"
+              style="height: 100%; width: 100%; border: 0px;"></iframe> -->
+          </div>
+        HTML
```

Comment:
This needs to be reverted to the original version as well because we don't have the iframe disabler in 0.26 which would enable this iframe in the front-end.

Reply:
Done in 03de7024f9b0d6748d64fb7b864aef94e764801a